scripts/docker/install_mysql.sh

```diff
@@ -75,7 +75,8 @@ install_mysql_client() {
         exit 1
     fi
 
-    export_key "467B942D3A79BD29" "mysql"
+    export_key "467B942D3A79BD29" "mysql-2022"
```

Comment:
I think it is no reason to import expired key, we can't use it legitimately without disable validation at all

Reply:
True.

Reply:
Fixed